src/Symfony/Component/VarDumper/Caster/StubCaster.php

```diff
@@ -89,4 +89,12 @@ public static function castScalar(ScalarStub $scalarStub, array $a, Stub $stub):
 
         return $a;
     }
+
+    public static function castVirtualStub(VirtualStub $virtualStub, array $a, Stub $stub): array
```

Comment:
we don't need this caster if VirtualStig extends ConstStub like UninitializedStub does